client/src/components/Avatar.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
```

Comment:
No need to add extra div - this can bring complications later on.
Custom components can always be component wrappers (start with custom component)